### PR TITLE
Fix some links and update some examples for Planck

### PR DESCRIPTION
**Pull request by agilecreativity**
Thanks for your great guide.
I have updated the following content to keep the guide more useful and up to date.

- Make the URL clickable and valid
- Improve some example for Planck using the latest guide 
- Misc improvement

Please feel free to edit or let me know if you like me to update anything else.

### CODE REVIEW

File: repls.org [hunk 1]
```diff
@@ -939,7 +938,7 @@
    [[https://github.com/cemerick/austin][Austin]] (apparently named after a [[https://en.wikipedia.org/wiki/The_Six_Million_Dollar_Man][sci-fi character from the 70's]]) is an
    alternative to the built-in browser REPL, trying to improve upon it in
    several ways. Since the project came out some other alternatives have come
-   onto the scene, notably [[Weasel][Weasel]] and [[Figwheel][Figwheel]], and unless you have specific
```
Comment (plexus) [inline repls.org]: These are internal links (try them on https://lambdaisland.com/guides/clojure-repls/clojurescript-repls). They link to another section, which then has a link to the project; please do not change them.

File: repls.org [hunk 2]
```diff
@@ -1038,7 +1037,7 @@
    based REPL (using the ~figwheel-sidecar~ library), and the piggieback
    middleware is loaded, then Figwheel will automatically do the right thing.
 
-   Figwheel uses WebSockets just like [[Weasel][Weasel]].
```
Comment (plexus) [inline repls.org]: Same as above.

**Review by plexus — CHANGES_REQUESTED** [review]
Thanks for the patches. There are a few links that should be left intact, otherwise it looks good.

File: repls.org (hunk 2 above)
Comment (agilecreativity) [reply repls.org]: Thanks @plexus 

It all make sense, I look at it from Github page which will not be linking correctly.
Thanks for pointing this out.

**Comment by plexus** [comment]
Thanks a lot!

**Pull request merged**